src/helpers/cache.ts

```diff
@@ -1,23 +1,22 @@
-const {
-  posix: { join },
-} = require('path')
+import { posix } from 'path'
+
+export const restoreCache = async ({ cache, publish }) => {
```

Comment:
I htink it's a good idea if we ensure that all exported functions have typed args and return values, even if they're just defined inline.